src/components/datePicker/header/datePickerHeader.tsx

```diff
@@ -0,0 +1,123 @@
+import { FC, ReactNode, useMemo } from 'react';
+import classNames from 'classnames/bind';
+import { Dropdown } from '@/components/dropdown';
+import { CalendarArrowIcon } from '@components/icons';
+import { DropdownOptionType, DropdownValue } from '@components/dropdown/types';
+import styles from './datePickerHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+const getYearsFrom = (start: number, amountYearsToGenerate = 20) => {
```

Comment:
Pls move to utils.
Is there any way to customize the `amountYearsToGenerate` argument?